data/korath/korath.txt

```diff
@@ -378,7 +378,6 @@ fleet "Korath Civilian Large"
 	variant 1
 		"Korath Chaser (World-Ship)"
 
-
```

Comment:
Why are you eating the whitespace between factions?  If this really is an issue, suggest it in stage I.  Where I'll still argue against it but at least it'll be in the right place.